src/integration/pages/featureIdxPage/crossPlatformTests.js

```diff
@@ -0,0 +1,32 @@
+import { runCommonCrossPlatformTests, runMainHeadingTests } from '../../common';
+
+export default service => {
+  runCommonCrossPlatformTests(service);
+  runMainHeadingTests();
+
+  describe('Sections', () => {
```

Comment:
Let's look at refactoring & reducing duplication, as these tests are used across FrontPages, IdxPages and now FeatureIdxPages